qml/AttributionButton.qml

```diff
@@ -40,16 +48,5 @@ IconButtonPL {
     width: icon.width
     z: 500
 
-    property string logo: ""
-
     onClicked: app.showMenu("AttributionPage.qml");
-    onLogoChanged: attributionButton.icon.source = logo ?
-        app.getIcon("icons/attribution/%1".arg(logo)) : "";
-
-    Connections {
-        target: app.styler
-        onIconVariantChanged: attributionButton.icon.source = attributionButton.logo ?
```

Comment:
`app.getIcon` depends on the iconVariant of the styler. This allows us to switch from light to the dark maps and change all icons accordingly. I expect that such binding rewrite will kill this property. Have you tried to swap between light/dark maps?

Reply:
Yes, this works. The iconvariant changes at the same time the logo does, so a logo change will always pick up the iconvariant.

Reply:
Just to be clear - its in some aspects a toss of a coin in this case. Its either we are strict and keep defined signals  (as before) in this case OR we write a comment regarding this in Map.qml. 

Taking into account that the application is already complex (while binding does reduce it!), I am inclined to be careful and go for the option 1 (keep signaling over here). What would be your call taking into account my arguments?